compositional/1D_COMP_NO_WELLS.DATA

```diff
@@ -0,0 +1,214 @@
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2024, SINTEF Digital, TNO
+
+-- This is a 1D, pressure-driven CO₂ flooding example involving three components and two phases.
+-- The components are CO₂, methane, and decane.
+-- The first and last cells are assigned very large pore volumes to emulate a source and sink,
+-- respectively, with constant pressure and compositions.
+
+------------------------------------------------------------------------
+RUNSPEC
+------------------------------------------------------------------------
+TITLE
+   SIMPLE CO2
+
+METRIC
+
+TABDIMS
+/
+OIL
+GAS
+DIMENS
+30 1 1
+/
+
+COMPS
+3 /
+
+-- FULLIMP /
+
+WELLDIMS
+2 2 1* 2/
+
+START
+  1 'JAN' 2016  /
+
+REGDIMS
+1 1 0 0 /
+
+UNIFOUT
+------------------------------------------------------------------------
+GRID
+------------------------------------------------------------------------
+INIT
+DX
+30*10
+/
+DY
+30*30
+/
+DZ
+30*1
+/
+
+TOPS
+30*0
+/
+
+
+PERMX
+30*100
+/
+
+PERMY
+30*100
+/
+
+PERMZ
+30*100
+/
+
+PORO
+1*1. 28*0.1 1*1.
+/
+------------------------------------------------------------------------
+EDIT
+------------------------------------------------------------------------
+
+MULTIPLY
+PORV 10000000 1 1 1 1 1 1/ --first gridblock
+PORV 10000000 30 30 1 1 1 1/ --last gridblock
+/
+------------------------------------------------------------------------
+PROPS
+------------------------------------------------------------------------
+
+CNAMES
+CO2
+METHANE
+DECANE
+/
+
+ROCK
+68.9476
+0
+/
+
+
+--RTEMP
+--150
+--/
+
+EOS
+PR /
+
+BIC
+0
+0
+0
+/
+
+ACF
+0.22394
+0.01142
+0.4884
+/
+PCRIT
+73.773
+45.992
+21.03
+/
+TCRIT
+304.128
+190.564
+617.7
+/
+MW
+44.00
+16.04
+142.28
+/
+
+VCRIT
+0.09412
+0.09863
+0.60980
+/
+
+SGOF
+-- Sg    Krg    Kro    Pcgo
+   0.0   0.0    1.0    0.0
+   0.1   0.1    0.9    0.0
+   0.2   0.2    0.8    0.0
+   0.3   0.3    0.7    0.0
+   0.4   0.4    0.6    0.0
+   0.5   0.5    0.5    0.0
+   0.6   0.6    0.4    0.0
+   0.7   0.7    0.3    0.0
+   0.8   0.8    0.2    0.0
+   0.9   0.9    0.1    0.0
+   1.0   1.0    0.0    0.0
+/
+
+
+STCOND
+15.0 1.0 /
+
+------------------------------------------------------------------------
+SOLUTION
+------------------------------------------------------------------------
+
+
+PRESSURE
+1*150 28*75. 1*37.5
+/
+SGAS
+30*1.
+/
+TEMPI
+30*150
+/
+ZMF
+1*0.99 29*0.5
+1*0.009 29*0.3
+1*0.001 29*0.2
+/
+
+
+
+------------------------------------------------------------------------
+SUMMARY
+------------------------------------------------------------------------
+--BPR
+--/
+
+------------------------------------------------------------------------
+SCHEDULE
+------------------------------------------------------------------------
+
+--RPTSCHED
+--'PRESSURE' /
+
+RPTRST
+BASIC = 1
```

Comment:
I'd prefer if this was `BASIC=2` instead.  BASIC=1 and BASIC=2 do have the same effect here, since we're using `UNIFOUT`, but `BASIC=2` is more explicit about that.